pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -158,24 +158,27 @@ func (r *RollbackREST) New() runtime.Object {
 	return &apps.DeploymentRollback{}
 }
 
-var _ = rest.Creater(&RollbackREST{})
+var _ = rest.NamedCreater(&RollbackREST{})
```

Comment:
Good catch. I expected this bug to affect more than the admission chain, but it seemed so.

Reply:
the integration test caught this as well... name wasn't getting populated in admission attributes so the webhook record step wasn't matching our expectations. turns out testing everything catches things :)

Reply:
Yeah, this test is great!